docker/InternVL_Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG CUDA_VERSION=cu12
+
+FROM openmmlab/lmdeploy:latest-cu12 AS cu12
+
+FROM openmmlab/lmdeploy:latest-cu11 AS cu11
+
+RUN python3 -m pip install timm flash-attn
```

Comment:
for cuda11.8, you need to install flash-attn from release pages eg. this may fail for `cu11` docker image
```
pip install  https://github.com/Dao-AILab/flash-attention/releases/download/v2.6.3/flash_attn-2.6.3+cu118torch2.1cxx11abiFALSE-cp310-cp310-linux_x86_64.whl
```

Reply:
> for cuda11.8, you need to install flash-attn from release pages eg. this may fail for `cu11` docker image
> 
> ```
> pip install  https://github.com/Dao-AILab/flash-attention/releases/download/v2.6.3/flash_attn-2.6.3+cu118torch2.1cxx11abiFALSE-cp310-cp310-linux_x86_64.whl
> ```

Don't worry,flash-attn will auto fit cuda and torch version after version 2.2.0 when compiling with pip;